contracts/contracts/Ramp.sol

```diff
@@ -513,6 +536,31 @@ contract Ramp is Ownable {
         emit MinDepositAmountSet(_minDepositAmount);
     }
 
+    /**
+     * @notice GOVERNANCE ONLY: Updates the sustainability fee. Fee updates are subject to a timelock in order to give users
```

Comment:
No timelock

Reply:
We aren't timelocking

Reply:
nvm thought you were saying it was missing, editing javadoc

Reply:
oops yeah, my comment was so vague